### ISSUE TITLE
where can I get old policy info of PPO agent

### ISSUE THREAD

**Issue by breadbread1984** [tensorflow/agents#460]
replay buffer for PPO agent requires old policy info for every time step. where I can get old policy logits for PPO agent at every time step if I dont use episode driver. thx

**Comment by sguada** [comment]
The `PPOPolicy` generates the needed information and add it to `policy_info` See
https://github.com/tensorflow/agents/blob/master/tf_agents/agents/ppo/ppo_policy.py#L235

**Comment by breadbread1984** [comment]
I create a PPOAgent object. I find that its policy of the agent is a greedy_policy object. therefore, the PolicyStep returned by agent.policy.action() has no policy_info. how I can have the PPOAgent object used PPOPolicy? thx
